legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-logical-plan/src/main/java/org/finos/legend/engine/persistence/components/planner/BitemporalDeltaPlanner.java

```diff
@@ -172,6 +172,17 @@ class BitemporalDeltaPlanner extends BitemporalPlanner
         }
     }
 
+    private Dataset getStagingDatasetWithoutDuplicates(Datasets datasets)
+    {
+        return datasets.stagingDatasetWithoutDuplicates().orElse(DatasetDefinition.builder()
+            .schema(stagingDataset().schema())
+            .database(stagingDataset().datasetReference().database())
```

Comment:
let's make sure we copy the name and alias as well - to avoid any test error later on

Reply:
Name and alias are generated at lines 181 and 182 - they have to be slightly different from the original stage table tho, as both of them will exist simultaneously